### PR TITLE
resolve issue with batch selects

### LINKED ISSUE #262: Replace multiple SELECT queries using execute_batch with execute
**Describe the bug**

In file database.py at function handle_hijack_update() we use psycopg2.extras.execute_batch to perform a mulitple selects.  

According to https://github.com/psycopg/psycopg2/issues/720 this is not supported and only the element from the last SELECT query is being returned.

**Affected Component(s)**
- [X] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

**To Reproduce**

**Expected behavior**

**Screenshots**

**System (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by pgigis**
### Description of PR

What component(s) does this PR affect?

- [X] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #262 

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by slowr** [comment]
:running: **Benchmark Results** :running:
- BGP Updates (updates/sec): **1739**
- Hijacks Detected (hijacks/sec): **149**

**Comment by pgigis** [comment]
The other option is to perform sequentially selects which I guess is even worse

File: backend/core/database.py [hunk 1]
```diff
@@ -589,32 +590,33 @@ def handle_hijack_update(self, message):
...
-                        with get_ro_cursor(self.ro_conn) as db_cur:
-                            psycopg2.extras.execute_batch(
-                                db_cur, query, (rekey_update_keys,), page_size=1000
+                        rekey_update_keys_lists = chunk_list(rekey_update_keys, 1000)
+                        for rekey_update_keys_list in rekey_update_keys_lists:
+                            query = (
+                                "SELECT key, prefix, origin_as, peer_asn, as_path, service, "
+                                "type, communities, timestamp FROM bgp_updates "
+                                "WHERE bgp_updates.handled=false AND bgp_updates.key IN "
+                                + str(tuple(rekey_update_keys_list))
```
Comment (vkotronis) [inline backend/core/database.py]: have you tested this? If it works, I totally agree with this approach.

Comment (pgigis) [reply backend/core/database.py]: yes, it works. However, codefactor does not agree. I am trying to find a workaround to make codefactor happy.

**Comment by pgigis** [comment]
Code updated. Codefactor is happy now :)

**Comment by vkotronis** [comment]
@pgigis please also update CHANGELOG before merging

**Pull request merged**